jsk_rosbag_tools/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(jsk_rosbag_tools)
+
+find_package(
+  catkin REQUIRED
+  catkin_virtualenv
+)
+
+catkin_python_setup()
+
+catkin_package(
+  CATKIN_DEPENDS
+)
+
+if (${catkin_virtualenv_VERSION} VERSION_LESS "0.6.1")
+  message(WARNING "Please install catkin_virtualenv>=0.6.1.")
+  message(WARNING "Current catkin_virtualen version is ${catkin_virtualenv_VERSION}")
+else()
+  catkin_generate_virtualenv(
+    INPUT_REQUIREMENTS requirements.in
+    PYTHON_INTERPRETER python3
+    USE_SYSTEM_PACKAGES TRUE
+    ISOLATE_REQUIREMENTS FALSE
```

Comment:
just out of curiosity, why we set FALSE?

Reply:
I no longer use catkin virtualenv.